src/scripts/components/dialog/view-info.js

```diff
@@ -0,0 +1,108 @@
+import React		from 'react/addons';
+import Carousel		from 'nuka-carousel';
+import Dialog		from '../../components/dialog';
+import TextBoxes	from './text-box';
+import Dropdown 	from '../dropdown';
+
+/**
+ *
+ */
+export default React.createClass({
+	mixins: [ Carousel.ControllerMixin ],
+	getInitialState(){
+		return { currentSlide: null }
+	},
+
+	componentDidMount() {
+		this.el = document.getElementById('application');
+		this.el.className = 'info-view-active';
+		this.avatar = document.getElementById('avatar');
+	},
+
+	componentWillUnmount() {
+		this.el.className = '';
+	},
+
+	componentDidUpdate(){
+		this.el.className =
+		`info-view-active slide-${this.state.carousels.carousel.state
+			.currentSlide}`;
+	},
+
+	render() {
+
+		let dropitems = [
+			{ icon: 'user',     content: 'Profile'  },
+			{ icon: 'language', content: 'Localization'  },
+			{ icon: 'bullhorn', content: 'Feedback'  },
+			{ icon: 'sign-out', content: 'Logout'  }
+		];
+
+		let objects = [
```

Comment:
You might want to document the structure of this array, so that people aren't left confused.